templates/bootstrap

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 cd $LAMBDA_TASK_ROOT
-./node-v{{NODE_VERSION}}-linux-x64/bin/node {{NODE_RUNNER_FILE}}
+/opt/node /opt/{{NODE_RUNNER_FILE}}
```

Comment:
Why this change? The whole goal of this is to allow for custom runtimes on Lambda. For example, Lambda doesn't support Node 10 yet. The whole goal of this package is to bundle everything needed to get a custom Lambda Node runtime. So you run it on your CI or something before deployment, and you can use any version of Node you want. Bundling requires a newer version of Node. But the runner file should be compatible with almost any Node version.

Reverting back to using the system Node version basically makes this package have no benefit.

Maybe I'm missing something here?

Reply:
/opt/node is just the node executable that is copied into the bundler from the build script. Instead of bundling the entire node download (docs, etc), you'll end up with a significantly smaller runtime bundle if you just copy the node executable from the download in the createRuntimeZip function of the package.

<img width="599" alt="screen shot 2019-02-11 at 8 22 31 am" src="https://user-images.githubusercontent.com/708797/52569119-1203c900-2dd6-11e9-97c9-fb82e5be3d4a.png">

Reply:
@gkurts This is the only place I'm seeing `/opt/node` in the PR tho. Am I missing something?